src/memex/search/index.py

```diff
@@ -92,25 +119,16 @@ class BatchIndexer(object):
     the search index.
     """
 
-    def __init__(self, session, es_client, request):
+    def __init__(self, session, es_client, request, target_index=None):
         self.session = session
         self.es_client = es_client
         self.request = request
 
         # By default, index into the open index
-        self._target_index = self.es_client.index
-
-    def index_all(self):
```

Comment:
Simply that it didn't really seem like there was any reason that the `BatchIndexer` needed to know about anything other than where it was indexing to.